lrc/cardano-sl-lrc.cabal

```diff
@@ -0,0 +1,59 @@
+name:                cardano-sl-lrc
+version:             0.1.0.0
+synopsis:            Cardano SL - Leaders and Rich men computation
```

Comment:
I suppose "richmen" is a term. "rich men" separately sounds funny to me. Because they're at least not "men" but "nodes" or "users".